core/src/main/scala/com/codacy/analysis/core/utils/HttpHelper.scala

```diff
@@ -26,7 +34,12 @@ class HttpHelper(apiUrl: String, extraHeaders: Map[String, String]) {
     } ++ extraHeaders
 
     val request: HttpRequest = dataOpt.map { data =>
-      Http(s"$remoteUrl$endpoint").postData(data.toString)
+      if (allowUnsafeSSL) {
+        Http(s"$remoteUrl$endpoint").options(HttpOptions.allowUnsafeSSL).postData(data.toString)
+      } else {
+        Http(s"$remoteUrl$endpoint").postData(data.toString)
+      }
```

Comment:
Same thing applied here.

Since it happens twice, can you create a private function, like:
```scala
private def httpOptions = if (allowUnsafeSSL) Seq(HttpOptions.allowUnsafeSSL) else Seq.empty
```

and then use it in both places with `.options(httpOptions)` ?
Thank you!